pkg/scans/checksrunner.go

```diff
@@ -290,6 +290,7 @@ func (c *ChecksRunner) CreateScanChecks(id string) error {
 			ID:                      scan.ID,
 			LastCheckCreated:        &lastCheck,
 			LastTargetCheckGCreated: &last,
+			TargetGroups:            &[]api.TargetsChecktypesGroup{},
```

Comment:
If I'm not wrong, we should do the same in the following cases:

https://github.com/adevinta/vulcan-scan-engine/blob/free-scan-table-space/pkg/scans/checksrunner.go#L165
https://github.com/adevinta/vulcan-scan-engine/blob/free-scan-table-space/pkg/scans/checksrunner.go#L177

Reply:
We could yes, If I recall correctly I did it on purpose because those cases don't happen often, on the contrary they should happen very rarely, so I though it would be better to leave the checks in the DB in order to have all the info for troubleshooting. Anyway it's not a strong opinion, so if taking into account the troubleshooting thing you still think it's worth it I'll do it. WDYT?

Reply:
All right, we can keep them for now and reevaluate in the future if it worth to keep these for troubleshooting.